hkmc2/shared/src/test/mlscript/nofib/mate.mls

```diff
@@ -557,22 +557,16 @@ fun testMate_nofib(dummy) =
     solve(bd, c, n)
 
 print(nofibListToString(testMate_nofib(0)))
-//│ >  - - - - - - - -
-//│ >  - - - - - - - -
-//│ >  - - - - - - - -
-//│ >  - - - - P - - -
-//│ >  - - - - p - k b
-//│ >  - P n - K - - -
-//│ >  - P - - - - - -
-//│ >  - q - - - - - -
-//│ > 
-//│ > White to move and mate in 3
-//│ > 
-//│ > 1. N/QB3-QR2,
-//│ >  if K-Q5; 2. B/KR4-KB2, K-QB5; 3. Q/QN1-KB1++
-//│ >  if K-Q7; 2. K-KB3, P/QN6xN/QR7; 3. B/KR4-K1++
-//│ >  if K-K7; 2. N/QR2-QN4,
-//│ >   if K-Q7; 3. Q/QN1-K1++
-//│ >   if K-K6; 3. Q/QN1-Q3++
-//│ >  if P/QN6xN/QR7; 2. Q/QN1-QB2, ...; 3. B/KR4-KB2++
-//│ > 
+//│ FAILURE: Unexpected runtime error
+//│ FAILURE LOCATION: mkQuery (JSBackendDiffMaker.scala:146)
+//│ ═══[RUNTIME ERROR] Error: ENOENT: no such file or directory, open 'hkmc2/shared/src/test/mlscript/nofib/input/heathcote3.prob'
```

Comment:
What happened? 🤔

Reply:
Oops, why doesn't the AllDiffTests report them. Let me check